eos/rare-b-decays/nonlocal-formfactors-p-to-p.cc

```diff
@@ -610,6 +635,14 @@ namespace eos
         return abs(this->_imp->nff->Hhat_plus(q2));
     }
 
+    template <typename Process_>
+    double
+    NonlocalFormFactorObservable<Process_, nff::PToP>::abs_normalized_plus(const double & q2) const
```

Comment:
Why the absolute value?

Reply:
It needs to be a double no? I also added imaginary and real parts